.github/workflows/.build-osx.yml

```diff
@@ -61,3 +60,24 @@ jobs:
         env:
           # GDAL currently doesn't build on m-chip, ref #93
           EXPECTED_DEPS_COUNT: "${{ matrix.runs-on == 'macos-15' && 11 || 12 }}"
+      - name: Generate tarball
+        run: |
+          mkdir -vp ${{ github.workspace }}/package
+          echo "# ${{ matrix.runs-on }} - TARBALL_NUMBERED '$TARBALL_NUMBERED'" >&2
+          echo "# ubuntu-focal - TARBALL_LATEST '$TARBALL_LATEST'" >&2
+          tar -C "$HOME" -czvf "$TARBALL_NUMBERED" install
+          cp -v "$TARBALL_NUMBERED" "$TARBALL_LATEST"
+        shell: bash
+        env:
+          TARBALL_NUMBERED: "${{ github.workspace }}/package/hyrax-dependencies-${{ matrix.runs-on }}-${{ github.run_number }}.tgz"
+          TARBALL_LATEST: "${{ github.workspace }}/package/hyrax-dependencies-${{ matrix.runs-on }}.tgz"
+      - name: Deploy to s3
+        # Will be a dryrun except for "push" triggers, which will only occur on
+        # main or master branch as defined by the workflow itself
+        run: |
+          aws s3 cp ${{ github.workspace }}/package s3://opendap.travis.build $aws_options
+        shell: bash
+        env:
+          access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
i haven't set these in github yet, so i expect this step to fail...i'm curious to see if the --dryrun option shows us what would happen regardless, htough.